sspl_test/coverage/coverage_setup.py

```diff
@@ -0,0 +1,172 @@
+#! /usr/bin/python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import shutil
+import os
+import pwd
+import signal
+import psutil
+import time
+import sys
+from cortx.utils.process import SimpleProcess
+# Add the top level directory to the sys.path to access classes
+topdir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+os.sys.path.insert(0, topdir)
+from framework.base.sspl_constants import DATA_PATH
+
+file_path = "/opt/seagate/cortx/sspl/low-level"
```

Comment:
what file path? Use some intuitive names. Also capital case for such constants
same for report_path

Reply:
addressed.